client/state/selectors/test/get-jetpack-onboarding-progress.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { JETPACK_ONBOARDING_STEPS as STEPS } from 'jetpack-onboarding/constants';
+import { getJetpackOnboardingProgress, isJetpackOnboardingStepCompleted } from 'state/selectors';
+
+describe( 'getJetpackOnboardingProgress()', () => {
+	test( 'should return onboading progress for the specified steps', () => {
+		const siteId = 2916284;
+		const state = {
+			jetpackOnboarding: {
+				settings: {
+					[ siteId ]: {},
+				},
+			},
+		};
+		const steps = [ [ STEPS.SITE_TITLE ], [ STEPS.SITE_TYPE ] ];
```

Comment:
Working on #21565, I just discovered that the brackets around `STEPS.SITE_TITLE  ` and `STEPS.SITE_TYPE` here are probably wrong/unneeded: They're array items, not object keys, so they will be interpreted as variable names, as they should.

Reply:
🤦‍♂️ 

PR coming in a bit.

Reply:
#21593, thanks Bernie